src/content/docs/en/guides/cms/decap-cms.mdx

```diff
@@ -8,13 +8,92 @@ i18nReady: true
 ---
 import Grid from '~/components/FluidGrid.astro'
 import Card from '~/components/ShowcaseCard.astro'
+import FileTree from "~/components/FileTree.astro"
 
 [Decap CMS](https://www.decapcms.org/) (formerly Netlify CMS) is an open-source, Git-based content management system.
+
+Decap allows you to take full advantage of all of Astro's features, including image optimization and content collections.
+
+Decap adds a route (typically `/admin`) to your project that will load a React app to allow authorized users to manage content directly from the deployed website. Decap will commit changes to your Astro project's source repository.
+
+## Installing DecapCMS
+
+There are two options for adding Decap to Astro:
+
+1. [Install Decap via a package manager](https://decapcms.org/docs/install-decap-cms/#installing-with-npm) with the following command:
+
+	```sh
+	npm install decap-cms-app --save
+	```
+ 
+1. Import the package into a `<script>` tag in your page `<body>`
+
+	```html title='/admin'
+    <body>
+      <!-- Include the script that builds the page and powers Decap CMS -->
+      <script src="https://unpkg.com/decap-cms@^3.1.1/dist/decap-cms.js"></script>
+    </body>
+	```
+ 
+
+## Configuration
```

Comment:
Check the indentation of this section. All the code samples etc. need to be indented in further!

Reply:
done!